src/components/custom/_signup-new.js

```diff
@@ -284,9 +291,20 @@ const SignupNew = ({
                 {localize('Create demo account')}
             </EmailButton>
             <Header as="p" type="small" weight="400" color="grey-5" mt="0.8rem">
-                {localize(
-                    'By signing up for a Deriv account, you agree to receive occasional updates about our products, services, and events. You can unsubscribe at any time in your account settings.',
-                )}
+                <Localize
+                    translate_text="By pressing “Create demo account”, you confirm that you are 18 or older. You understand that we may use your email address to send you information about Deriv products and services as well as market news. You can always unsubscribe from these emails in your account settings. For more information, please take a look at Deriv’s <0>Security and privacy</0>."
+                    components={[
+                        <StyledLocalizedLink
+                            key={0}
+                            to="/tnc/security-and-privacy.pdf"
+                            size="12px"
```

Comment:
```suggestion
                            size="1.2rem"
```
A small change to support for screens > 1980px for consistency with the `Header` sizes.

Reply:
Sure thing!